assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -51,10 +51,12 @@ function useUploadVideoFrame({ updateMediaElement }) {
         id: posterId,
         source_url: poster,
         media_details: { width: posterWidth, height: posterHeight },
-      } = await uploadFile(obj);
+      } = await uploadFile(obj, {
+        post: id,
+        media_source: ['editor', 'poster-generation'],
```

Comment:
Why set both? `poster-generation` alone should suffice, and `media_source` can stay an enum

Reply:
So the reason for the editor term in the first place, so we workout where an image was uploaded. Technically editor and post generation for both true here. If you don't see the value in adding both, then I will remove it and make this PR simpler.

Reply:
Yeah I think they're partially redundant. Let's remove it and make it simpler 👍

Reply:
Removed in https://github.com/google/web-stories-wp/pull/5635/commits/8b50b112f965e4683190fe01791c2c96c9e3b5ab